code/game/machinery/doors/brigdoors.dm

```diff
@@ -351,6 +351,11 @@
 	name = "Cell 6"
 	id = "Cell 6"
 
+/obj/machinery/door_timer/tactical_pet_storags

```

Comment:
Just noticed this thing's a .dm file, as in Polaris. Might want to do a _vr addition?